bob.yaml

```diff
@@ -25,5 +23,3 @@ dependencies:
   - docker
   - gcc
 
-use-nix: true
-nixpkgs: https://github.com/NixOS/nixpkgs/archive/5857574d45925585baffde730369414319228a84.tar.gz
```

Comment:
Isn't it beneficial to pin the exact nixpks version? I would actually recommend to do that. Or Not?!